includes/fbbot/class-omise-fbbot-conversation-generator.php

```diff
@@ -0,0 +1,209 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_Conversation_Generator' ) ) {
+  return;
+}
+
+class Omise_FBBot_Conversation_Generator {
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function greeting_message( $sender_id ) {
+		$greeting_message = Omise_FBBot_Message_Store::get_greeting_message( $sender_id  );
+		$buttons = Omise_FBBot_Message_Store::get_default_menu_buttons();
+		
+		return FB_Button_Template::create( $greeting_message, $buttons );
+	}
+
+	public static function helping_message() {
+		$helping_message = Omise_FBBot_Message_Store::get_helping_message();
+		$buttons = Omise_FBBot_Message_Store::get_default_menu_buttons();
+
+		return FB_Button_Template::create( $helping_message, $buttons );
+	}
+
+  public static function rechecking_order_number_message() {
+    $rechecking_order_massage = Omise_FBBot_Message_Store::get_rechecking_order_number_message();
+    return FB_Message_Item::create( $rechecking_order_massage );
+  }
+
+  public static function get_ordet_status_message( $order_id ) {
+    $order_status = Omise_FBBot_WooCommerce::check_order_status( $order_id );
+
+    if ( ! $order_status ) {
+      $message = FB_Message_Item::create( __( "Sorry, your order number not found. Can you try to check it again ? :'(", 'omise' ) );
+      return $message;
+    }
+
+    $message = FB_Message_Item::create( sprintf( __( "BAMM! Your order status is '%s' :]", 'omise' ), $order_status ) );
+
+    return $message;
```

Comment:
@igroomgrim just minor, but you can directly return `FB_Message_Item::create(...)` without assigning a result to `$message` variable.